images/L-Dev-NorthwindPullSave/L-Dev-NorthwindPullSave/Generic Data Pull.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Lab Developer: Categories Pull and Save"
```

Comment:
Shouldn't this be in a different folder location? Also this page is not in the _data/toc.yaml file

Reply:
Also this page is in the wrong location